addons/mail/models/mail_mail.py

```diff
@@ -182,7 +182,7 @@ def _postprocess_sent_message(self, success_pids, failure_reason=False, failure_
                     messages = notifications.mapped('mail_message_id').filtered(lambda m: m.is_thread_message())
                     # TDE TODO: could be great to notify message-based, not notifications-based, to lessen number of notifs
                     messages._notify_message_notification_update()  # notify user that we have a failure
-        if not failure_type or failure_type in ['RECIPIENT', 'mail_missing']:  # if we have another error, we want to keep the mail.
+        if not failure_type or failure_type in ['mail_email', 'mail_missing']:  # if we have another error, we want to keep the mail.
```

Comment:
Commit message states "m_mail" but here it seems to be "mail_email" (same for others apparently).